### PR TITLE
Allow disabling autosweep on nodes

### PR DESCRIPTION
**Pull request by RodriFS**

### CODE REVIEW

File: src/Jobs/SweepNodeWalletsJob.cs [hunk 1]
```diff
@@ -148,9 +148,17 @@ public async Task Execute(IJobExecutionContext context)
             _logger.LogError("{JobName} failed on node with id: {NodeId}, reason: node not found",
                 nameof(SweepNodeWalletsJob),
                 managedNodeId);
-            throw new ArgumentException("node not found", nameof(node));
+            await context.Scheduler.DeleteJob(context.JobDetail.Key, context.CancellationToken);
+            return;
```
Comment (RodriFS) [inline src/Jobs/SweepNodeWalletsJob.cs:152]: jobs weren't cancelled when a node was deleted so it kept giving this error forever

File: src/Pages/Nodes.razor [hunk 2]
```diff
@@ -179,7 +179,7 @@
 
     private async Task GetData()
     {
-        _nodes = await NodeRepository.GetAll();
+        _nodes = await NodeRepository.GetAllManagedByFundsManager();
```
Comment (RodriFS) [inline src/Pages/Nodes.razor:182]: Not managed nodes should not be visible in the Nodes view, but they should be selectable from the channel request view

File: src/Jobs/SweepNodeWalletsJob.cs (hunk 1 above)
Comment (Jossec101) [reply src/Jobs/SweepNodeWalletsJob.cs:152]: Nice catch

**Review by Jossec101 — APPROVED** [review]
LGTM

**Pull request merged**